py3status/modules/online_status.py

```diff
@@ -21,13 +20,7 @@
 @author obb
 """
 
-import os
-import subprocess
-try:
-    # python3
-    from urllib.request import urlopen
-except:
-    from urllib2 import urlopen
+from urllib.parse import urlsplit
```

Comment:
`sudo python2 setup.py develop` in your py3status dir to install.  you may need to install other  modules eg requests for python2 too as they keep their libs independently.